### ISSUE TITLE
kind request for some accessibility

### ISSUE THREAD

**Issue by jazoni100** [microsoft-search/pnp-modern-search#494]
Well, Modern Sharepoint at this time does not meet even nearly European Web Accessibility directive (WCAG 2.0). But it does not mean Modern Search Webparts couldn't. 

For starters some suggestions:

Search Box Webpart
- Search button should have aria-label="search" attribute, screen reader users don't get any information what this button does.

Search results Webpart
- result count could have role="status" attribute to inform screen reader users about results.

Search pagination
- Should have role="navigation" attribute (or use <nav> element) and aria-label="pagination" attributes on the container
- pagination links are focusable with tab navigation but dont't have focus ring with screen reader.
- page numbers could have aria-label="page 1" for clarification.
- next page and last page buttons don't have any information for screen readers, use aria-label attributes for these link.

Search filters
- should have a title or aria-label on container to inform screen reader users these are filters.
- filter titles are clickable with screen reader but don't have any information what clicking does, should at least have role="button" and aria-expanded attributes. 
- filter titles are not focuseble with tab navigation. Use tabindex="0" to make non links or buttons focusable.

There's probably much more. But this for start.

Thank you!

**Comment by wobba** [comment]
Hi,
this is a great point, and depending on how you use the parts I see the issue. If you only use them for roll-up, then you can handle all of this in the handlebar template. If using search box and filters, they might be missing this.

If someone wants to take a pass an implement this we'll happily merge it.

**Comment by wobba** [comment]
@jazoni100 feel free to create a new one for latest v4 to address issues on accessibility.
